doc/api_reference/variables.rst

```diff
@@ -306,62 +329,141 @@ Optional module variables
 Static Libraries
 ^^^^^^^^^^^^^^^^
 
-.. _required-staticlib-variables:
+Static library repositories have two possible modes of use: building the static library archive
+from source, and linking the static library (with any additional sources) into an application. Most
+static library variables in the XCommon CMake API are used in just one of these two modes of use.
 
-Required static library variables
-"""""""""""""""""""""""""""""""""
+.. _archive-staticlib-variables:
 
-The same as the :ref:`required-module-variables`, and also:
+Variables for archive build
+"""""""""""""""""""""""""""
 
-``XMOS_SANDBOX_DIR``
-  The path to the root of the sandbox directory. This is only required if ``LIB_DEPENDENT_MODULES``
-  is non-empty. This must be set in the static library's ``CMakeLists.txt`` file before including
-  ``lib_build_info.cmake``. See :ref:`sandbox-structure`.
+``LIB_ARCHIVES``
+  List of archives to build. If not set, a single archive is built and is named by the ``LIB_NAME``
+  variable. Example:
```

Comment:
single archive (per supported architecture) is built?

Reply:
Good clarification.